### ISSUE TITLE
Add the ability to specify an interface

### ISSUE THREAD

**Issue by mhriemers** [crazy-max/ddns-route53#616]
When using a dual WAN setup in a load balancing configuration, it is unknown over which interface the request will go. This can be achieved using a custom Transport, as explained [here](https://stackoverflow.com/questions/30552447/how-to-set-which-ip-to-use-for-a-http-request).

I can produce a pull-request for this around the coming weekend.

**Comment by crazy-max** [comment]
@mhriemers Sure SGTM!

**Comment by mhriemers** [comment]
Alright, what would be the best way to implement this in your opinion? Since the IPv4 and IPv6 addresses are gathered only once and reused for all records, I'm thinking about an extra configuration item on the root (such as `Interface`) or a command flag. WDYT?

**Comment by crazy-max** [comment]
@mhriemers I think we can extend the configuration with a `global` section like:

```yaml
global:
  interface: foo

credentials:
  accessKeyID: "ABCDEFGHIJKLMNO123456"
  secretAccessKey: "abcdefgh123456IJKLMN+OPQRS7890+ABCDEFGH"

route53:
  hostedZoneID: "ABCEEFG123456789"
  recordsSet:
    - name: "ddns.example.com."
      type: "A"
      ttl: 300
    - name: "ddns.example.com."
      type: "AAAA"
      ttl: 300
    - name: "another.example2.com."
      type: "A"
      ttl: 600
```

In a follow-up we could also override the global interface for specific records.

**Comment by crazy-max** [comment]
Fixed in #901
